docker/setup-dev.sh

```diff
@@ -0,0 +1,28 @@
+
+#!/bin/bash
+set -e
+
+# Supabase Local Development Setup Script
+# Usage: source .env && ./setup-dev.sh
+
+echo "[1/4] Exporting environment variables from .env..."
+if [ -f .env ]; then
+  source .env
+else
+  echo "No .env file found. Please create one from .env.example."
+  exit 1
+fi
+
+echo "[2/4] Starting Docker Compose services..."
+docker compose -f docker-compose.yml -f dev/docker-compose.dev.yml up -d
+
+echo "[3/4] Waiting for containers to become healthy..."
+docker compose -f docker-compose.yml -f dev/docker-compose.dev.yml ps
+
+echo "[4/4] Checking health of core services..."
+for service in supabase-db supabase-rest supabase-auth supabase-studio supabase-analytics supabase-storage; do
```

Comment:
Steps 3 and 4 should be combined in a single loop that waits for all the containers to be healthy (with a timeout in case there's a problem).